File: Programming Basics/JavaScript/Nested loops/Lab/building.js
function solve(input) {
    const floors = Number(input[0]);
    const rooms = Number(input[1]);

    for (let i = floors; i > 0; i--) {
        let letter;
        if (i == floors) letter = "L";
        else if (i % 2 == 0) letter = "O";
        else letter = "A";

        let result = "";
        for (let j = 0; j < rooms; j++) {
            if (j > 0) result += " ";
            result += `${letter}${i}${j}`;
        }

        console.log(result);
    }
}

File: Programming Basics/C++/Nested loops/Labs/Building/Building.cpp
#include <iostream>
using namespace std;

int main()
{
    int floors, rooms;
    cin >> floors >> rooms;

    for (int i = floors; i > 0; i--)
    {
        char letter;
        if (i == floors) letter = 'L';
        else if (i % 2 == 0) letter = 'O';
        else letter = 'A';

        for (int j = 0; j < rooms; j++)
        {
            if (j > 0) cout << ' ';
            cout << letter << i << j;
        }

        cout << endl;
    }
}
